try-cb-dotnet/wwwroot/index.html

```diff
@@ -0,0 +1,6 @@
+<h1>.NET Travel Sample API</h1>
+    A sample API for getting started with Couchbase Server and the .NET SDK.
+    <ul>
+        <li><a href="/apidocs">Learn the API with Swagger, interactively</a>
+        <li><a href="https://github.com/couchbaselabs/try-cb-dotnet">GitHub</a>
+    </ul>
```

Comment:
Probably good to have newline here.

Reply:
At the end of the file, you mean? I suspect I'm using the VSCode defaults, is that worth changing?

Reply:
yeah, I always set it to automatically add a newline at the end of a file, just because I forget all the time 😄

Reply:
ok, updated settings, force pushed.